### PR TITLE
Fix show bubble for unsupported languages

### PR DESCRIPTION
**Pull request by atuchin-m**
Resolves https://github.com/brave/brave-browser/issues/19300

Disable the UI language filtering when `UseBraveTranslateGo` is disabled.
This resolve issues when we don't show the bubble when the target/sources language is unsupported by brave.

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: browser/translate/brave_translate_browsertest.cc [hunk 1]
```diff
@@ -352,34 +352,54 @@ class BraveTranslateBrowserDisabledFeatureTest
 
 IN_PROC_BROWSER_TEST_F(BraveTranslateBrowserDisabledFeatureTest,
                        FeatureDisabled) {
-  EXPECT_CALL(backend_request_, Call(_)).Times(0);
-  ResetObserver();
-  ASSERT_TRUE(ui_test_utils::NavigateToURL(
-      browser(), embedded_test_server()->GetURL("/espanol_page.html")));
-  WaitUntilLanguageDetermined();
+  // Set target language to FR, that it's unsupported target language
```
Comment (iefremov) [inline browser/translate/brave_translate_browsertest.cc:355]: that is unsupported

**Review by iefremov — APPROVED** [review]

**Review by goodov — APPROVED** [review]

File: browser/translate/brave_translate_browsertest.cc (hunk 1 above)
Comment (atuchin-m) [reply browser/translate/brave_translate_browsertest.cc]: Fixed

**Comment by stephendonner** [comment]
Verified `PASSED` using 

Brave | 1.34.4 Chromium: 96.0.4664.35 (Official Build) nightly (x86_64)
-- | --
Revision | 8df3babf8da8faa8aacb78f4848ef66a93b4944b-refs/branch-heads/4664@{#680}
OS | macOS Version 11.6.1 (Build 20G224)

Used the testplan from https://github.com/brave/brave-browser/issues/19300#issue-1048190885

deutschland.de | aol.de | inap.co.jp
---------|----------|---------
Screen Shot 2021-11-11 at 1 31 29 PM | Screen Shot 2021-11-11 at 1 31 38 PM | Screen Shot 2021-11-11 at 1 32 28 PM

**Comment by kjozwiak** [comment]
@atuchin-m the above landed in `master` which is currently `1.34.x` so it needs to stay in that milestone. The only thing that's moved into `1.33.x` is https://github.com/brave/brave-browser/issues/19300 as the above fix is currently in `1.33.x` via https://github.com/brave/brave-core/pull/11007.

**Pull request merged**